product/compare/hosts.yaml

```diff
@@ -28,7 +28,7 @@ include:
     group: Properties
     columns:
     - cpu_total_cores
-    - memory_mb
+    - ram_size_in_bytes
```

Comment:
I think the reports are good the way they were

Reply:
drift state uses one of these to determine the column names and i think it'll end up reporting 0 for something that has ram_size_in_bytes set, that's why i made this change